modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css

```diff
@@ -0,0 +1,19 @@
+.bar-chart{
+  flex-wrap: wrap;
```

Comment:
```suggestion
  <div className="align-items-center bar-chart d-flex flex-wrap justify-content-between">
```

Reply:
Bootstrap flex-wrap doesn't work for this class.